### ISSUE TITLE
refactor and create results class with methods

### ISSUE THREAD

**Issue by sanjaynagi** [sanjaynagi/AnoPrimer#46]
The structure of AnoPrimer is very simple - its a single python script with functions only, which output variables which can then be provided as inputs to other functions. It was my first python package I had written at the time, and two years on, I think it could be structured differently, which would make it neater more useful. 

In particular, I think it would be nice if running Primer3 within AnoPrimer produced a class, which then had primer data, primer3 input parameters, and further methods associated with it, such as looking in ag3 for variation, summarising ag3 cohorts, plotting locations of primers, blatting against the PEST genome, and writing to file. 

If I do this, Ill also need to update the notebooks, and the PAMCA-malariaGEN course.

**Comment by sanjaynagi** [comment]
It's not quite clear what the name should be for the class, both in the package itself and the variable name when using it. In the package, it could be `AnoPrimerResults`.

Usage could be like:

```
primers = AnoPrimer.designPrimers() 
primers.plot_primer_snp_frequencies()
```
?

**Comment by sanjaynagi** [comment]
Perhaps we should also split the scripts up, to have 

design.py 
evaluate.py
utils.py

**Comment by sanjaynagi** [comment]
Closed with #47 v2.0.0 anoprimer
